Sources/Architecture/X86/HardwareBreakpointManager.cpp

```diff
@@ -227,7 +227,6 @@ ErrorCode HardwareBreakpointManager::enableDebugCtrlReg(uint64_t &ctrlReg,
       EnableBit(ctrlReg, infoIdx + 3);
       break;
     case 8:
-      DS2LOG(Warning, "8-byte hw breakpoints not supported on all devices");
```

Comment:
Might have asked this question a long time ago: Why is this duplicate? I only see this warning once, do we also warn elsewhere?